.github/workflows/slsa-builder-go.yml

```diff
@@ -179,63 +225,67 @@ jobs:
           path: ${{ steps.build-gen.outputs.go-binary-name }}
           retention-days: 5
 
+      #TODO(upload): verify how the action handles malicious names.
+      - name: Upload the provenance
+        uses: actions/upload-artifact@82c141cc518b40d92cc801eee768e7aafc9c2fa2 # v2.3.1
+        with:
+          name: ${{ steps.build-gen.outputs.provenance-unsigned }}
```

Comment:
I think this will fail, it should be `steps.sha256-gen...`? Maybe also rename the name of the step to say it's computing the hadh of the binary and prov

Reply:
done